octopusdeploy_framework/resource_process_step.go

```diff
@@ -621,3 +640,9 @@ func mapActionExecutionPropertiesToState(properties map[string]core.PropertyValu
 
 	return valuesMap, diags
 }
+
+// isBooleanLikeString checks if a string represents a boolean value
+func isBooleanLikeString(s string) bool {
```

Comment:
Why need a custom function over `strconv.ParseBool()` ?

(Not your change btw 😄 )